### DISCUSSION TITLE
Hello everyone, does the slint-viewer have a UI with DesignMode and style switching as previewed in LSP?

### DISCUSSION THREAD

**Discussion by me0106** [slint-ui/slint#5267] [Q&A]
lsp preview:
![image](https://github.com/slint-ui/slint/assets/22869648/76099fb1-ef46-45a7-8ca2-aad2e1db7862)
------
slint-viewer: 
![image](https://github.com/slint-ui/slint/assets/22869648/3462636a-3f05-40b4-89af-4326a50a7d85)

**Comment by hunger** [comment]
No, that functionality is built into the slint-lsp binary only. Sorry.

We actually use the Language Server protocol to ask the editor to do all the text changes, so we really need to have that connected editor.
